### ISSUE TITLE
Adding support to python3.8

### ISSUE THREAD

**Issue by Daniel-ChenJH** [tusharsadhwani/yen#17]
Hi there! I ran ‘yen list’ and found that only 4 versions of python are supported here. I wonder if python 3.8 can be supported later? Thanks.
```Available Pythons:
3.12.3
3.11.9
3.10.14
3.9.19
```

**Comment by tusharsadhwani** [comment]
Hey, I've been meaning to support all the versions down to 3.7. It's possible to support a lot more specific minor and patch versions as well.

I'll do that.

**Comment by Daniel-ChenJH** [comment]
thanks. I have got the latest version of yen 0.4.3, as i can see in PyPi, and installed by pip install yen. However, I cannot see py38 in the output of 'yen list'.

![image](https://github.com/tusharsadhwani/yen/assets/79645566/1d33c054-843a-453c-a257-c2adf5354399)


```
cjh@DESKTOP-NERJPTB:~/domain/DeepKE$ pip install yen
Collecting yen
  Using cached yen-0.4.3-py3-none-any.whl.metadata (2.7 kB)
Requirement already satisfied: microvenv>=2023.2.0 in /home/cjh/anaconda3/lib/python3.11/site-packages (from yen) (2023.5.post1)
Requirement already satisfied: rich>=13.5.3 in /home/cjh/anaconda3/lib/python3.11/site-packages (from yen) (13.7.1)
Requirement already satisfied: markdown-it-py>=2.2.0 in /home/cjh/anaconda3/lib/python3.11/site-packages (from rich>=13.5.3->yen) (2.2.0)
Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /home/cjh/anaconda3/lib/python3.11/site-packages (from rich>=13.5.3->yen) (2.15.1)
Requirement already satisfied: mdurl~=0.1 in /home/cjh/anaconda3/lib/python3.11/site-packages (from markdown-it-py>=2.2.0->rich>=13.5.3->yen) (0.1.0)
Using cached yen-0.4.3-py3-none-any.whl (32 kB)
Installing collected packages: yen
Successfully installed yen-0.4.3
cjh@DESKTOP-NERJPTB:~/domain/DeepKE$ yen create venv -p 3.8
Error: requested Python version is not available. Use 'yen list' to get list of available Pythons.
cjh@DESKTOP-NERJPTB:~/domain/DeepKE$ yen list
Available Pythons:
3.12.3
3.11.9
3.10.14
3.9.19
cjh@DESKTOP-NERJPTB:~/domain/DeepKE$ 
```

can you explain why no accessible python 3.8.18 can be used here? Thanks for any help.

**Comment by tusharsadhwani** [comment]
This was a bug, and it should now be fixed in `v0.4.4`. Can you check and confirm?

**Comment by Daniel-ChenJH** [comment]
thanks! py3.8.18 is accessible now!
